pcm.h

```diff
@@ -6,19 +6,19 @@ class PcmOutput {
     virtual void flush() {}
 };
 
-PcmOutput *makePcmOutputWav(const char *fn);
+PcmOutput *makePcmOutputWav(const char *fn, int rate);
 
 #ifdef unix
-PcmOutput *makePcmOutputUnix();
-#define makePcmOutputAudio makePcmOutputUnix
+PcmOutput *makePcmOutputUnix(int rate);
+#define makePcmOutputAudio(rate) makePcmOutputUnix(rate)
```

Comment:
It's not necessary to include the parameter list in this macro. It can just read

    #define makePcmOutputAudio makePcmOutputUnix

and the parameter list will take care of itself.